packages/embedder/ai_models/TextGenerationInference.ts

```diff
@@ -0,0 +1,88 @@
+import {truncate} from 'fs'
```

Comment:
Note: this file was renamed from TextGenerationInterface to TextGenerationInference; for some reason when I wrote this over xmas I kept read "Inference" as "Interface". I've also renamed this repo: https://github.com/ParabolInc/node-llama-cpp-text-generation-inference